packages/react/src/components/header/components/headerActionBar/HeaderActionBar.tsx

```diff
@@ -155,7 +155,10 @@ export const HeaderActionBar = ({
         </div>
       </div>
       <HeaderLanguageSelector fullWidthForMobile />
-      <HeaderActionBarNavigationMenu />
+      <HeaderActionBarNavigationMenu
+        logo={logoProps?.href ? <span className={styles.logoWrapper}>{logo}</span> : logo}
```

Comment:
If it has a href it will get wrapped in a link and the positioning of the img gets all fckd up 😄 This is actually a duplication of the logo that is always visible.